submissions/TaniaBondarenko/a-tiny-js-world-update/index.js

```diff
@@ -0,0 +1,75 @@
+class Inhabitant {
+  constructor(name, gender, saying, friends) {
+    this.name = name;
+    this.gender = gender;
+    this.saying = saying;
+    this.friends = friends;
+  }
+  toString() {
+    return (` I\'m <b>${this.name}</b>, a ${this.gender}. To my friend ${this.friends} I always say: "${this.saying}"`);
+  }
+};
+
+class Person extends Inhabitant {
+  constructor(name, gender, saying, friends, species='human', legs=2, hands=2) {
+    super(name, gender, saying, friends)
+    this.species = species;
+    this.legs = legs;
+    this.hands = hands;
+  }
+  toString() {
+    return `Since I'm <em>${this.species}</em>,I have ${this.legs} legs and ${this.hands} hands.` + super.toString();
+  } 
+};
+
+class Animal extends Inhabitant {
+  constructor(name, gender, saying, species, friends, legs=4, hands=0) {
+    super(name, gender, saying, friends )
+    this.species = species;
+    this.legs = legs;
+    this.hands = hands;
```

Comment:
Why do we need `hands` for animals that do not have hands by natural design?

Reply:
Done